index.d.ts

```diff
@@ -68,7 +69,58 @@ declare namespace Eris {
   type MessageContent = string | AdvancedMessageContent;
   type MFALevel = 0 | 1;
   type PossiblyUncachedMessage = Message | { channel: TextableChannel | { id: string; guild?: Uncached }; guildID?: string; id: string };
-  type InteractionType = 1 | 2;
+
+  // Interaction
+  type InteractionDataOptions = {
+    name: string;
+    type: Constants["CommandOptionTypes"][keyof Constants["CommandOptionTypes"]];
+    value?: Constants["CommandOptionTypes"][keyof Constants["CommandOptionTypes"]];
```

Comment:
`value` should be string
```suggestion
    value?: string;
```

Reply:
*wait yea why did I change that*